README.md

```diff
@@ -1,2 +1,316 @@
-# PSC2023-Capstone
- This repository is for an fictisious conference on my country. This is the final assigment of module 1 named Capstone Project.
+<style>
+  @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700;800&display=swap');
+
+  /*Import box icons*/
+  @import url('https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css');
+
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+</style>
+
+<a id="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="images/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+<!--
+  <h3><b>Microverse README Template</b></h3>
```

Comment:
- [ ] From the requirements given, We should remove the Microverse logo in our README and if we have our own logo, we can replace it with it. Please remove it 😃 👍